styles/styles.css

```diff
@@ -613,6 +594,25 @@ main .section.layout-60-40 .section-container > div:nth-child(2) {
     h5 {
       font-size: var(--h5-font-size-desktop);
     }
+
+    main .section.layout-60-40 .section-container {
```

Comment:
Are we using the `60` and `40` values anywhere? As an author, I get an impression that I can also do things like `50-50` or `70-30` which likely wouldn't work.  How about calling this a `side-by-side` section variant?

Reply:
I had started with side-by-side only then we had a discussion about more same kind of layout ..like you said 50-50 , 70-30, that's why we move this name so that in future in the same naming convention more style could be created.

Reply:
ok likely I missed that conversation else would have supported your case of using just `side-by-side` 😄 .
Anyhow, what's the expectation if one added > 2 blocks in a section marked as 60-40?

Reply:
Also please ensure we add this section variation in the block library.

Reply:
>what's the expectation if one added > 2 blocks in a section marked as 60-40?
first two block would be shown side-by-side and first will take 60% and other will get 40% width. 
@sdmcraft